### PR TITLE
Refine README

### PR DESCRIPTION
**Pull request by ascendbruce**
## Why is this change necessary?

- Friendly landing page
- Instructions for install/uninstall

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,68 @@
...
+
+## Benefits
+
+* No more painful spring cleaning in Downloads folder.
+* No extra actions need to be taken.
+* Save disk space.
+
+## How it works
+
+* It moves old files into Trash once a day. (old file: added into Downloads folder more than 24 hours ago)
```
Comment (aranair) [inline README.md]: Can just combine into one sentence, might be clearer:

`It moves old files that have been added into Downloads folder more than 24 hours ago`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,68 @@
...
+* Still work with slept macs. It will run right after waking up.
```
Comment (aranair) [inline README.md]: Still works with Macs that are in sleep mode.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,68 @@
...
+* The only thing you need to do is to move useful files to a safe place soon after downloaded. The rest will be deleted within 24~48 hours automatically. (You can restore them from trash bin though)
```
Comment (aranair) [inline README.md]: `... move useful files to a safe place after download.`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,68 @@
...
 
 ## Install
 
+We recommend you to do a (last) spring cleaning before installing. Otherwise all old downloads will just be thrown into trash regardlessly.
```
Comment (aranair) [inline README.md]: ... will be moved to trash.

**Review by aranair — APPROVED** [review]
lgtm situp

**Comment by ascendbruce** [comment]
LOL

**Pull request merged**